src/components/MovieCard.js

```diff
@@ -1,11 +1,36 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import './MovieCard.css';
 
-const MovieCard = () => {
+const MovieCard = props => {
```

Comment:
Do we need to pass props here if we're using Redux?  I apologize that all my comments are questions!